code/clement/html_css_flask/lab04/templates/index.html

```diff
@@ -0,0 +1,233 @@
+
+    <!-- ---------------Devicon------------------------------- -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@v2.15.1/devicon.min.css">
+
+    <title>Personal Portfolio</title>
+
+</head>
+
+<body>
+    <!-- -------------Navbar Sections------------- -->
+    <nav class="navbar navbar-expand-lg navbar-light">
+        <div class="container-fluid">
+            <a class="navbar-brand" href="#">
+                <img src="https://c8.alamy.com/zooms/9/42a17f39eee8418daeaadc1d5ce359ba/2ba53jx.jpg" alt="">
+            </a>
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup"
+                aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse d-lg-flex justify-content-end" id="navbarNavAltMarkup">
+                <div class="navbar-nav mb-2 mb-lg-0">
+                    <a class="nav-link active" aria-current="page" style="color: aliceblue;" href="#">Home</a>
+                    <a class="nav-link active" aria-current="page" style="color: aliceblue;" href="#">About
+                        Me</a>
+                    <a class="nav-link active" aria-current="page" style="color: aliceblue;" href="#">Skills</a>
+                    <a class="nav-link active" aria-current="page" style="color: aliceblue;" href="#">Portfolio</a>
+                    <a class="nav-link active" aria-current="page" style="color: aliceblue;" href="#">Contact
+                        Me</a>
+                    <button class="btn btn-primary">Login</button>
+                </div>
+            </div>
+        </div>
+    </nav>
+    <!---------- Information About Me----------- -->
+    <div class="about-me mt-5">
+        <div class="col-11 mx-auto">
+            <div class="row">
+                <div class="col-md-6">
+                    <img class="about-img" src="{{url_for('static', filename='images/IMG-20170420-WA0003.jpg')}}"
+                        alt="">
+                </div>
+                <div class="col-md-6">
+                    <p class="myname">
+                    <h1 class="display-1 mb-5 mt-2">Clement Buadu</h2>
+                        </p>
+                        <p class="text mb-5 display-5">
+                            Full-Stack Developer in <span>MD</span>
+                        </p>
+                        <p class="h3 mb-5">
+                            I'm a developer focusing front & back end developing, creating user-friendly Websites. I'm
+                            passionate about building an excellent software that improves and brings happeners to the
+                            lives of others around the world....
+                        </p>
+                        <div class="line"></div>
+                        <!-- <section class="row mt-3">
```

Comment:
Commented code is great for development. Before deploying to production remove comments to reduce load time on lower-end devices.